src/Engine/Manager/RenderManager.cpp

```diff
@@ -237,15 +255,18 @@ void RenderManager::Render(World& world, const glm::mat4& translationMatrix, con
                 Entity* entity = mesh->entity;
                 // If entity does not have material, it won't be rendered.
                 if (entity->GetComponent<Material>() != nullptr) {
-                    renderer->DepthRenderStaticMesh(mesh->geometry, viewMatrix, projectionMatrix,  entity->GetModelMatrix());
+                    renderer->DepthRenderStaticMesh(mesh->geometry, viewMatrix, projectionMatrix, entity->GetModelMatrix());
                 }
             }
         }
     }
+    }
+    }
 
     // Render static meshes.
-    {
-        PROFILE("Render static meshes");
+    { PROFILE("Render static meshes");
+    { GPUPROFILE("Render static meshes", Video::Query::Type::TIME_ELAPSED);
+    { GPUPROFILE("Render static meshes", Video::Query::Type::SAMPLES_PASSED);
         renderer->PrepareStaticMeshRendering(viewMatrix, projectionMatrix);
         for (Mesh* mesh : meshComponents) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The function 'GetDirectionalLights' is never used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)